README.md

```diff
@@ -11,8 +11,11 @@
 
 [Netlify Labs](https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/) gives you early access to new features before they’re available to everyone. We welcome your feedback, and we encourage you to join the [Netlify research program](https://www.netlify.com/research-program/) to help us continuously improve the platform. Experimental features are works-in-progress, so you may find some bugs along the way.
 
-## Features in Labs
+## Features currently in Labs
 
 * [Netlify Graph](features/graph/documentation/)
+* [Scheduled Functions](features/scheduled-functions/documentation/README.md)
+
+## Features graduated from Labs
+
 * [Command Palette](features/command-palette/)
```

Comment:
[sand] Instead of linking to the labs docs here, I think it would make sense to link to the docs site and the latest version of the docs.

```suggestion
* [Command Palette](https://docs.netlify.com/welcome/command-palette/)
```